controllers/storagecluster/storageclient.go

```diff
@@ -87,10 +88,10 @@ func (s *storageClient) ensureDeleted(r *StorageClusterReconciler, storagecluste
 	return reconcile.Result{}, nil
 }
 
-func (s *storageClient) updateClientConfigMap(r *StorageClusterReconciler, namespace string) error {
+func (s *storageClient) updateClientConfigMap(r *StorageClusterReconciler, storageCluster *ocsv1.StorageCluster) error {
```

Comment:
Prefer sending the individual flags and not the entire storagecluster type
```suggestion
func (s *storageClient) updateClientConfigMap(r *StorageClusterReconciler, namespace string, useHostNetwork bool ) error {
```